getClass() on Annotation

I'm building Swagger using the [Bazel build system](https://bazel.build/). Code in ParameterProcessor.java fails on some additional checks automatically invoked by Bazel when building Java code:

```
vendor/github.com/swagger-api/swagger-core/modules/
    swagger-core/src/main/java/io/swagger/util/ParameterProcessor.java:287: error: 
    [GetClassOnAnnotation] Calling getClass() on an annotation may  return a proxy class
                         rsDefault = (String) item.getClass().getMethod("value").invoke(item);
                                                           ^
     (see http://errorprone.info/bugpattern/GetClassOnAnnotation)
   Did you mean 'rsDefault = (String) item.annotationType().getMethod("value").invoke(item);'?
```

I'm not a Java developer, so I'm not even trying to understand if there is a downside to calling `annotationType()` here in place of `getClass()`.

However, for my invocations of the generator in order to generate a Go, a PHP and a Python client library, updated code seems to still generate the correct output. I've done nothing except apply the suggestion provided in the output.

Turning off the warnings also allows me to build the project, but that seems like a bad approach.

Thoughts?

--- a/modules/swagger-core/src/main/java/io/swagger/util/ParameterProcessor.java
+++ b/modules/swagger-core/src/main/java/io/swagger/util/ParameterProcessor.java
@@ -376,7 +376,7 @@ public class ParameterProcessor {
                     apiParam = new ApiImplicitParamWrapper((ApiImplicitParam) item);
                 } else if ("javax.ws.rs.DefaultValue".equals(item.annotationType().getName())) {
                     try {
-                        rsDefault = (String) item.getClass().getMethod("value").invoke(item);
+                        rsDefault = (String) item.annotationType().getMethod("value").invoke(item);
                     } catch (Exception ex) {
                         LOGGER.error("Invocation of value method failed", ex);
                     }

